AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminOpenAdobeStockSearchModalActionGroup.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
According to the best practices in MFTF we should use one action group per file.

Reply:
@rogyar For example in action group AdminCatalogSearchTermActionGroup we have several action groups.

Reply:
Yes, that's true. Unfortunately, the tests that have been created in the very beginning are not the best example of best practices. From that moment we are trying to improve the best practices considering mistakes made in the past. It's like underscores for private/protected variables in the initial version of the Magento 2 core codebase and other similar examples. You can check the following wiki for short description of the best practices https://github.com/magento/magento-functional-tests-migration/wiki/Best-practices